ipaserver/rpcserver.py

```diff
@@ -784,6 +785,46 @@ def marshal(self, result, error, _id=None,
         return dump.encode('utf-8')
 
 
+class jsonserver_i18n_messages(jsonserver):
+    """
+    JSON RPC server for i18n messages only.
+    """
+
+    key = '/i18n_messages'
+
+    def __init__(self, api):
+        super(jsonserver_i18n_messages, self).__init__(api)
```

Comment:
You don't have to provide a super call if you just call the super method without additional call.

Reply:
Perhaps, i misunderstand your thoughts. Excuse me, please.
super() is used for indirect reference as well as delegating method calls to some base class in the instance’s ancestor tree in case of further inheritance.

Reply:
Since the ``__init__`` method doesn't perform additional work, you can drop the entire ``__init__`` method from the class. Python automatically calls the method from the parent class.